walkthroughs/howto-outlier-detection/README.md

```diff
@@ -398,3 +398,96 @@ All resources created in this walkthrough can be deleted via:
 ```bash
 ./deploy.sh delete && ./mesh.sh down
 ```
+
+## Part 2: Using Virtual Gateway
+
+Note: This part lists the changes to Service Mesh, ECS Service and the commands (which are different) from the above steps when using a Virtual Gateway.
+
+With the Ingress Gateway we no longer need a frontend application. VirtualGateway resource would act as the ColorGateway instead of a VirtualNode. The Virtual Gateway allows resources outside your mesh to communicate to resources that are inside your mesh.
+## Setting up Service Mesh with Virtual Gateway.
+
+Our mesh contains one virtual gateway, `front-vg` and  a virtual node `color-node`. The virtual gatewayb`front-vg` routes to a virtual service `color.howto-outlier-detection.local` that is provided by the `color-node`.
+The actual services mirror this setup with a frontend service calling a color service backend. There is a single frontend service task and four color service tasks. We will send requests through an ALB pointing to the frontend service.
+
+```bash
+./mesh.sh up-vg-setup
+```
+
+## Deploy Service
+
+  We'll build the color applications under `src` into Docker images, create and push to the ECR repo under the account `AWS_ACCOUNT_ID`, then deploy CloudFormation stacks for network infrastructure, the bastion host, and the ECS services.
```

Comment:
So this is essentially a standalone application, vs updating the previous example?

Reply:
yes, sort of, I have kept the previous example as it is. I have added the other example with Virtual Gateway. We are re-using the docker images and the infrastructure stack from the previous example. We are deploying new ECS services and creating a new mesh with virtual gateway for part 2. (as per the latest commit).